### PR TITLE
Move artifact building instructions into docs

### PR DESCRIPTION
**Pull request by omus**

### CODE REVIEW

File: docs/src/building-artifacts.md [hunk 1]
```diff
@@ -6,6 +6,38 @@ For Ray.jl releases we provide pre-built binary artifacts to allow for [easy ins
...
+   export GITHUB_TOKEN
+   julia --project upload_tarballs.jl
+   ```
+
+5. Once all assets are uploaded, checkout a new branch (based off of `main`) and run the `bind_artifacts.jl`. This will modify the local `Artifacts.toml` with the artifact(s) associated with the Ray.jl version specified in the Project.toml. After running this you should commit and push the changes to a new PR.
+
+   ```sh
+   julia --project bind_artifacts.jl
+   ```
+   <!-- TODO: The CI workflows for this PR can run successfully and the referenced artifacts will be accessible. -->
```
Comment (glennmoy) [inline docs/src/building-artifacts.md:38]: can remove this now that we have https://github.com/beacon-biosignals/Ray.jl/pull/152
```suggestion
```

File: docs/src/developer-guide.md [hunk 2]
```diff
@@ -20,10 +20,10 @@ Building the Ray.jl project requires the following tools to be installed. This l
 4. Navigate to the root of the Ray.jl repo
 5. Install [Ray dependencies](https://docs.ray.io/en/latest/ray-contribute/development.html#preparing-to-build-ray-on-macos):
 
-```sh
-brew update
-brew install bazelisk wget
-```
+   ```sh
```
Comment (glennmoy) [inline docs/src/developer-guide.md:23]: are indented snippets just a personal preference?

**Review by glennmoy — APPROVED** [review]

File: docs/src/developer-guide.md (hunk 2 above)
Comment (omus) [reply docs/src/developer-guide.md:23]: These were always intended to be a part of the list number above. I just forgot the syntax as it's odd to have a 3-space indentation.

Comment (glennmoy) [reply docs/src/developer-guide.md:23]: ah I see!

**Pull request merged**